src/Lazy/entries.ts

```diff
@@ -0,0 +1,27 @@
+type Entries<T extends Record<string, any>, K extends keyof T> = K extends any
+  ? [K, T[K]]
+  : never;
+
+/**
+ *
+ * Creates an array of own enumerable string keyed-value pairs.
```

Comment:
How about this?

Creates Iterable/AsnycIterable 
or Returns Iterable/AsyncIterable